m365-developer-proxy-abstractions/GraphBatchResponsePayload.cs

```diff
@@ -18,7 +18,7 @@ public class GraphBatchResponsePayloadResponse {
     [JsonPropertyName("body")]
     public dynamic? Body { get; set; }
     [JsonPropertyName("headers")]
-    public Dictionary<string, string>? Headers { get; set; }
+    public List<Dictionary<string, string>>? Headers { get; set; }
```

Comment:
It seems odd to define headers as a list of dictionaries. Shouldn't it be a list of key-value pairs instead?

Reply:
Using a Dictionary (which is in effect the same thing AFAIK), means we can use Linq to do "ToDictionary()"

Reply:
This has now changed to a List<KeyValuePair<string,string>> due to the dictionary complicating some other areas.